src/index.js

```diff
@@ -59,7 +59,7 @@ class EdidReader {
   // Linux fetch EDID
   getLinuxSystemEdids() {
     // /sys/devices/pci0000\:00/0000\:00\:02.0/drm/card0/card0-HDMI-A-1/edid
-    return glob('/sys/devices/pci*/0000:*/drm/card*/card*/edid')
+    return glob('/sys/devices/pci*/0000:*/*/drm/card*/card*/edid')
```

Comment:
For local usage I added changes you've added to fix the error, however this line causes glob to not match any files, not sure if you added it for a reason, but I removed it and only then it matched the files needed

Reply:
Changing this made it work on Ubuntu for me back then so… :man_shrugging: Maybe `…/0000:**/drm/card…` would work for both of us?